### ISSUE TITLE
Training data folder contents

### ISSUE THREAD

**Issue by pranathivemuri** [AllenInstitute/aics-ml-segmentation#5]
Hi!

I am trying to use this repo and couldn't find enough documentation about how the d[atafolder, a key in the config file should be organized. ](https://github.com/AllenInstitute/aics-ml-segmentation/blob/master/configs/train_config.yaml#L45)
1. What images should it have and how should they be organized, can you please point me to some documentation regarding that
2. After going around the repo, I have found that the ground truth masks are saved in the same folder as the images as GT_ome.tiff files, and there is a cost mapper image that should be saved. Can you please explain to me the contents of the cost mapper file and its data type. Is it the probability map of the classes in the mask?
3. Also, can you please explain how the sizeIn, sizeOut, and outputCh were set in your config file ?  I have looked at this documentation https://github.com/AllenInstitute/aics-ml-segmentation/blob/master/docs/doc_train_yaml.md but I couldn't figure out what p and kp are in the equation

Thanks a lot!

**Comment by pranathivemuri** [comment]
@jxchen01 Any input would be appreciated!

**Comment by jxchen01** [comment]
Thanks for your interest. I would be more than happy to help.

Regarding to your question 1 and 2, you may take a look at this page https://github.com/AllenInstitute/aics-ml-segmentation/blob/master/docs/bb2.md#take-all

Regarding to your question 3, the table in that page already pre-computes the sizeIn and sizeOut for you under different GPU memory. If your GPU memory is very different, or you would like to try something different, as stated in the doc, p and z can be any positive integers that make size_out has all positive values. 

Here is a sample yaml file https://github.com/AllenInstitute/aics-ml-segmentation/blob/master/configs/train_config.yaml

Let me know if you have more questions or if there is anything else I can help.

Thanks.

**Comment by jxchen01** [comment]
Also, if you have more questions, feedback, or any requests, feel free to leave a comment on our allen cell forum:  

https://forum.allencell.org/c/software-code
